activemq-karaf/src/main/resources/features-core.xml

```diff
@@ -55,6 +56,7 @@
       <bundle dependency="true">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.jasypt-spring31/1.9.2_1</bundle>
       <bundle dependency="true">mvn:org.apache.servicemix.specs/org.apache.servicemix.specs.stax-api-1.0/${servicemix.specs.version}</bundle>
       <bundle dependency="true">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.xpp3/${xpp3-bundle-version}</bundle>
+      <bundle dependency="true">mvn:joda-time/joda-time/${jodatime-bundle-version}</bundle>
```

Comment:
I'd wonder if this is actually needed (like the zookeeper, doesnt seem like it is should actually be related to the API change even if it was in the commit), and so again assuming it isnt would also probably omit it or add a second commit removing it (and related property) again too.

Reply:
Same

Reply:
Yeah I'm removing it, seems like it shouldn't be there